### PR TITLE
Add template to allow env vars to change URLs

### PR DESCRIPTION
**Pull request by PhilWun**
#### Short Description
This pull request adds a template for store.module.ts. This allows environment variables to be substituted into the file to change the URLs in the administration tab.

#### Proposed Changes
  * add template for store.module.ts
  * change Dockerfile to substitute environment variables into the template

### CODE REVIEW

File: src/app/store/store.module.ts.template [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2018 University of Stuttgart.
```
Comment (lharzenetter) [inline src/app/store/store.module.ts.template:2]: I guess we should update this to be 2018-2021.

**Review by lharzenetter — CHANGES_REQUESTED** [review]
Except comment LGTM!

File: src/app/store/store.module.ts.template (hunk 1 above)
Comment (PhilWun) [reply src/app/store/store.module.ts.template]: Done.

File: src/assets/config.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "API_ENDPOINT_HOST": "",
```
Comment (lharzenetter) [inline src/assets/config.json:2]: I guess we should add the defaults here? I.e., `localhost`, `1337`, and `8080`?

Comment (PhilWun) [reply src/assets/config.json:2]: `config.json` gets overwritten when the container starts, but it needs to be there if you start the UI in a way other than Docker. That's why the default values are in `store.module.ts`.

**Pull request merged**